client/components/jetpack/daily-backup-status/status-card/backup-realtime-message.tsx

```diff
@@ -0,0 +1,83 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+
+type Props = {
+	backupDate: number; // timestamp
+	eventsCount: number;
+};
+
+export const BackupRealtimeMessage: FunctionComponent< Props > = ( {
+	backupDate,
```

Comment:
We need as a parameter the date of the backup selected and the date of the base backup